server/sensorAlerts.js

```diff
@@ -0,0 +1,94 @@
+// Third-party dependencies
+const { t } = require('i18next')
+
+// In-house dependencies
+const { CHATBOT_STATE, helpers } = require('brave-alert-lib')
+const db = require('./db/db')
+
+let braveAlerter
+
+function setup(braveAlerterObj) {
+  braveAlerter = braveAlerterObj
+}
+
+async function handleAlert(location, alertType) {
```

Comment:
I agree; and it can be unit tested! + It matches the buttonAlerts.js file in BraveButtons, so that's prepared for a future codebase merge. I don't believe I changed much, maybe added an extra comment or two. The new code working with numberOfAlerts is all new, though :)